module/VuFind/src/VuFind/Cover/Loader.php

```diff
@@ -435,31 +436,14 @@ protected function fetchFromAPI()
             $this->contentType = 'image/jpeg';
             $this->image = file_get_contents($this->localFile);
             return true;
-        } elseif (isset($this->config->Content->coverimages)) {
-            $providers = explode(',', $this->config->Content->coverimages);
-            foreach ($providers as $provider) {
-                $provider = explode(':', trim($provider));
-                $apiName = strtolower(trim($provider[0]));
-                $key = isset($provider[1]) ? trim($provider[1]) : null;
-                try {
-                    $handler = $this->apiManager->get($apiName);
-
-                    // Is the current provider appropriate for the available data?
-                    if ($handler->supports($ids)) {
-                        if ($url = $handler->getUrl($key, $this->size, $ids)) {
-                            $success = $this->processImageURLForSource(
-                                $url, $handler->isCacheAllowed(), $apiName
-                            );
-                            if ($success) {
-                                return true;
-                            }
-                        }
-                    }
-                } catch (\Exception $e) {
-                    $this->debug(
-                        get_class($e) . ' during processing of ' . $apiName
-                        . ': ' . $e->getMessage()
-                    );
+        } else {
+            $urls = $this->getCoverUrls();
+            foreach ($urls as $url) {
+                $success = $this->processImageURLForSource(
+                    $url['url'], $url['handler']->isCacheAllowed, $url['apiName']
```

Comment:
I think this should be `isCacheAllowed()`, not just `isCacheAllowed`.

Reply:
You are right, I'll fix it